src/core/CompactionQueue.java

```diff
@@ -86,6 +86,8 @@
 
   /** If this is X then we'll flush X times faster than we really need.  */
   private final int flush_speed;  // multiplicative factor
+  
+  private long compactedKVTimestamp;
```

Comment:
Can't have it here as the compaction class is called from multiple threads. This works for just the scheduled compaction but the ``compact()`` method is called asynchronously.